ChildMindInstitute/mindlogger-admin-OLD-Vue
# Add support for string token values in dashboard #345

Add support for string token values in the token logger dashboard.

Issue #345 

Testing this PR:
1. Create a new user response in a token applet using the latest mobile app version
2. Log into the admin panel using the owner or reviewer account
3. Check whether the dashboard loads correctly or not
